index.html

```diff
@@ -1334,38 +1398,42 @@ <h2>The <dfn>NDEFRecordType</dfn> enum</h2>
       <th>NDEF record type</th>
       <th>NDEFRecord recordType</th>
       <th>NDEFRecord mediaType</th>
-      <th>NDEFRecord data</th>
+      <th>NDEFRecord data getters</th>
```

Comment:
As these are functions, I am not sure we should call them getters. but I am unsure about this